netmon.cc

```diff
@@ -467,13 +486,18 @@ CheckResult PINGChecker::perform()
 
     SConnect(sock, s);
     SSetsockopt(sock, SOL_IP, IP_RECVTTL, 1);
+    if (d_dontFragment) {
```

Comment:
Is this going to work for IPv6 as well?

Reply:
As i mentioned in manual, it is noop in IPv6, at least i verified in Linux 6.7.4. But just to be on safe side i added additional verification, to apply this flag on IPv4 only, so it will be future proof, as it is more undefined behavior right now.